controllers/interfaces.go

```diff
@@ -95,6 +101,7 @@ type CombinedDoguManager interface {
 	deleteManager
```

Comment:
Jeez, what kind of large interface is this? This isn't your fault specifically but the whole team's because... WE ALREADY have segregated interfaces like the `exportManager`. 

![Simpson meme: Someone holding a piece of newspaper which show an image of granpa Simpson. Head title: Old man yells at cloud](https://github.com/user-attachments/assets/cc9c8be2-76c3-4155-911c-88211be6e8a7)